feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -79,9 +79,9 @@ export class CustomerTicketingService implements CustomerTicketingFacade {
     return [GetTicketsQueryResetEvents];
   }
 
-  protected createTicketEventCommand: Command<TicketEvent, unknown> =
-    this.commandService.create<TicketEvent>(
-      (ticketEvent) =>
+  protected createTicketEventCommand: Command<TicketEvent, TicketEvent> =
+    this.commandService.create<TicketEvent, TicketEvent>(
```

Comment:
Could you please explain me the double TicketEvent?

Reply:
this is part of the code that came from the original branch and should not appear here as something that I added